extensions/file-cutter/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "file-cutter",
+  "title": "File Cutter",
+  "description": "Cut and paste files in Finder",
+  "icon": "cut-icon.png",
+  "author": "rokartur",
+  "categories": [
+    "Productivity",
+    "System",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "cut-files",
+      "icon": "cut-icon.png",
+      "title": "Cut files",
+      "subtitle": "Cut selected files in Finder",
```

Comment:
Subtitle should not be used as a description - consider making it your title instead